### PR TITLE
[Meshery] Renamed perforance managed API endpoints

### LINKED ISSUE #779: [Meshery] Restructure performance management related API endpoints
**Current Behavior**
The collection of performance related API endpoints is here - https://github.com/layer5io/meshery/blob/master/router/server.go#L34-L38. These can be grouped behind a common path.

**Desired Behavior**
These need to be renamed and placed behind a common `/perf` path. 

**Acceptance Criteria**
The performance endpoints should be like so:

```
/api/perf/load-test
/api/perf/load-test-smps
/api/perf/load-test-prefs
/api/perf/results
/api/perf/result
```

### PR DESCRIPTION
**Pull request by chandrashritii**
Signed-off-by: Shriti <shritichan@gmail.com>

**Description**

- Restructured all performance management related API endpoints behind a common /perf path.
- Covered all references of the API endpoints in mesheryctl and Meshery UI.

This PR fixes #779 

#### Checklist
- [x] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [x] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

File: router/server.go [hunk 1]
```diff
@@ -33,11 +33,11 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 	mux.Handle("/api/k8sconfig/ping", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.KubernetesPingHandler))))
 	mux.Handle("/api/mesh/scan", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.InstalledMeshesHandler))))
 
-	mux.Handle("/api/load-test", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestHandler))))
-	mux.Handle("/api/load-test-smps", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestUsingSMPSHandler))))
-	mux.Handle("/api/load-test-prefs", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestPrefencesHandler))))
-	mux.Handle("/api/results", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.FetchResultsHandler))))
-	mux.Handle("/api/result", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetResultHandler))))
+	mux.Handle("/api/perf/load-test", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.LoadTestHandler))))
```
Comment (kushthedude) [inline router/server.go:36]: This just changes the routes on the server while as on UI the prefix perf is not added. @leecalcote Do we just need a change in routes at server level or at the user level too? As currently, there is no change in routes for a user who is accessing Meshery through UI.

Comment (leecalcote) [reply router/server.go:36]: @kushthedude of the UI code affected here, it all seems to haven updated. From the tests that I did, the this updates worked.

Comment (leecalcote) [reply router/server.go:36]: @kushthedude I understand what you mean now - not the API endpoints, but the UI routes. Got it. Thank for you diligently accounting for these as well. Let's give further thought to those as new UI pages come forth.

Comment (kushthedude) [reply router/server.go:36]: Cool!

**Pull request merged**